web-client/src/transaction.rs

```diff
@@ -837,6 +846,14 @@ pub struct PlainSetActiveStakeData {
     pub new_active_balance: u64,
 }
 
+/// JSON-compatible and human-readable format of set inactive stake data.
```

Comment:
Copy-pasta
```suggestion
/// JSON-compatible and human-readable format of retire stake data.
```

Reply:
Please also fix the comment for `PlainSetActiveStakeData`, to read "set active stake data".

Reply:
Fixed! thank you so much for the good eye🕵️‍♂️